backend/worker/worker.go

```diff
@@ -1238,6 +1238,20 @@ func (w *Worker) RefreshMaterializedViews(ctx context.Context) {
 		log.WithContext(ctx).Fatal(e.Wrap(err, "Error refreshing daily_session_counts_view"))
 	}
 
+	if err := w.Resolver.DB.Transaction(func(tx *gorm.DB) error {
```

Comment:
should we run the two refreshes concurrently wrt each other?

Reply:
I think it's better to run them consecutively only because they each cause a large IOPS spike, more likely to cause resource contention if they're running concurrently

Reply:
sounds good!